[Project Submission] braid

```diff
@@ -0,0 +1,56 @@
+
```

Comment:
Could you please change this file to Markdown File?

Reply:
Sorry about that -- added .md extension

Reply:
@longfeiWan9 review requested please

Reply:
@ufkhan97 Thank you for the update. Please also move it to the right folder, which is `community/projects/application/`.

Reply:
Moved!